core/src/main/java/jenkins/security/ApiTokenProperty.java

```diff
@@ -209,23 +209,27 @@ public boolean matchesPassword(String token) {
      */
     private boolean hasPermissionToSeeToken() {
         // Administrators can do whatever they want
-        if (SHOW_LEGACY_TOKEN_TO_ADMINS && Jenkins.get().hasPermission(Jenkins.ADMINISTER)) {
+        return checkPermission(SHOW_LEGACY_TOKEN_TO_ADMINS, user);
+    }
+
+    private static boolean checkPermission(boolean showLegacyTokenToAdmins, User user) {
+        if (showLegacyTokenToAdmins && Jenkins.get().hasPermission(Jenkins.ADMINISTER)) {
```

Comment:
This variable name is no longer indicative of its use because of its two similar but differing use-cases

Reply:
`allowAdmin`?

Reply:
Yeah i think that seems ok

Reply:
I've gone w/ `trustAdmins` https://github.com/jsoref/jenkins/pull/3
If people are ok w/ that, I'll merge it into this PR (probably Thursday)

Reply:
Fine to me